war/src/main/less/modules/notifications.less

```diff
@@ -0,0 +1,135 @@
+.jenkins-notification {
+  position: fixed;
+  left: 1.2rem;
+  bottom: 1.2rem;
```

Comment:
The pop-up may get lost in the bottom left corner, I'd say, depending on the view you are at.

Did you try to keep the center position with the new animation?

Reply:
it looks nice where it currently is to me

Reply:
could possibly be in the top right?

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/43062514/188517458-01619349-bb2e-4cbb-bb95-60869a4e85dc.png">

<img width="1440" alt="Screenshot 2022-09-05 at 23 27 47" src="https://user-images.githubusercontent.com/43062514/188517422-f0bf77cc-9909-4ce2-962b-1636c7f772ce.png">

Here it is in the top center/right. I'm not too sure how I feel about it up there, it all feels quite busy at the top and it might prevent a user from clicking a control/nav bar.

How would the bottom right sound? It's probably the emptiest corner in Jenkins.

Reply:
Sure let’s try that

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/43062514/189097893-ab48801f-d5b5-4ad5-9d2b-fb2f8363b90d.png">

Here it is in the bottom right, I definitely prefer that to the top imo - it's a lot less busy. 

What are your thoughts guys?

Reply:
what if it's in the top but always below the header?

Reply:
Bottom left, next to save and apply buttons, sounds less noisy to me.

Reply:
> what if it's in the top but always below the header?

I think that might introduce complexities, e.g. having to account for the visible portion of the header if the user is slightly scrolled down the page.

> Bottom left, next to save and apply buttons, sounds less noisy to me.

Are you both okay with bottom left then? We can always tweak it in future once page layout changes start coming in if needs be.